### PR TITLE
Doc reorg

### PR DESCRIPTION
**Pull request by mnunberg**

### CODE REVIEW

File: content/sdk/c/lcb-pars-shared.dita [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
+<topic id="toplevel">
+  <title></title>
+  <body>
+    <p id="connstr-intro"> You can specify additional options when connecting to the cluster by
+            using the <i>connection string</i>. The connection string indicates to the client where
+            cluster nodes may be found and how to connect to them. The <i>connection string</i> is
+            common to other Couchbase SDKs as well as the <xref
+                href="../webui-cli-access.dita#concept_j3z_zhm_zs">command-line client</xref>. The
+                <i>connection string</i> uses a URI-like format and is similar to a <i>DSN</i> used
```
Comment (ingenthr) [inline content/sdk/c/lcb-pars-shared.dita]: We had this conversation.  Despite wikipedia calling them equivalent, DSN has a specific meaning and I don't think we should propagate incorrect this incorrect understanding.

**Comment by mnunberg** [comment]
@ingenthr I pushed a new commit here. Can you please merge this, or otherwise re-edit the content to your liking?

File: content/sdk/c/lcb-pars-shared.dita [hunk 2]
```diff
@@ -8,7 +8,6 @@
             cluster nodes may be found and how to connect to them. The <i>connection string</i> is
             common to other Couchbase SDKs as well as the <xref
                 href="../webui-cli-access.dita#concept_j3z_zhm_zs">command-line client</xref>. The
-                <i>connection string</i> uses a URI-like format and is similar to a <i>DSN</i> used
-            in other database systems.</p>
+                <i>connection string</i> uses a URI-like syntax</p>
```
Comment (ingenthr) [inline content/sdk/c/lcb-pars-shared.dita]: I'm already editing it.  You've left this one without a full stop.

Comment (mnunberg) [inline content/sdk/c/lcb-pars-shared.dita]: So much for editing with vim :)

**Comment by ingenthr** [comment]
Unfortunately, your editing of the PHP managing connections and my review/edits conflicted in such a way that it's hard to see the intraline diff.  I can probably work it out by going back to before I edited, but that's not the highest priority at the moment so I've just taken my changes.  It was a big change since it converted that concept to a topic.

Other than that, the DSN discussion and some review I made on the stability section (which we should discuss) it was all rebased, reviewed, incorporated.

**Pull request closed without merging**